docker/etc/trex_cfg.yaml

```diff
@@ -7,3 +7,4 @@
                    default_gw : 242.2.2.2
                  - ip         : 242.2.2.2
                    default_gw : 241.1.1.1
+  latency_measurement: "nanoseconds"
```

Comment:
[Usually other parameters are placed above port_info](https://github.com/codilime/trex-core/blob/86bfcbf7695fe301671bd01d426ff181e2206653/scripts/cfg/cfg_example1.yaml)

Reply:
I know, but I chose to distinguish this one as it is “ours”

Reply:
Fair enough